Week-2/assignment-1/scss/style.css

```diff
@@ -0,0 +1,306 @@
+}
+
+.footer-container {
+  width: 100vw;
+  background-color: #a55eea;
+  overflow-x: hidden;
+}
+
+.footer-container .footer-layout {
+  width: 1440px;
+  margin: 0px auto;
+  background-color: #a55eea;
+}
+
+.footer-container .footer-box {
+  max-width: 1250px;
+  margin: 0px auto;
+  padding: 40px 0;
+  background-color: #a55eea;
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: 1fr 1fr;
+      grid-template-columns: 1fr 1fr;
+  place-items: center;
+  color: #fff;
+}
+
+@media only screen and (max-width: 550px) {
+  #parent {
+    width: 100vw;
+    min-height: 500px;
+    background-color: #f7f7f7;
+    overflow-x: hidden;
+  }
+  .layout-container {
+    width: 100%;
+    margin: 0px auto 0 auto;
+    background-color: #f7f7f7;
+    overflow-x: hidden;
+  }
+  .hero-container {
+    width: 97%;
+    margin: 0px auto;
+    padding: 40px 0;
+  }
+  .hero-container h1 {
+    font-size: 33px;
+    margin-top: 20px;
+    text-align: center;
+    color: #a55eea;
+  }
+  .hero-container .description {
+    font-size: 20px;
+    margin-top: 20px;
+    text-align: center;
+    color: #585858;
+  }
+  .hero-container .article {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 1fr;
+        grid-template-columns: 1fr;
+    -ms-grid-rows: 540px 1fr;
+        grid-template-rows: 540px 1fr;
+    grid-gap: 10px;
+    margin: 20px 0 30px 0;
+    place-items: center;
+  }
+  .hero-container .article .box1 {
+    width: 95%;
+  }
+  .hero-container .article .box1 p {
+    font-size: 18px;
+    line-height: 1.3;
+  }
+  .hero-container .article .box1 img {
+    border-radius: 10px;
+    width: 100%;
+    margin-top: 20px;
+  }
+  .hero-container .article .box2 {
+    width: 95%;
+  }
+  .hero-container .article .box2 article:nth-child(1) {
+    margin: 0px 0;
+  }
+  .hero-container .article .box2 article {
+    margin: 5px 0;
+  }
+  .hero-container .article .box2 article h5 {
+    color: #343434;
+    font-size: 20px;
+    line-height: 1.3;
+    margin-bottom: 10px;
+  }
+  .hero-container .article .box2 article p {
+    font-size: 17px;
+    line-height: 1.3;
+  }
+  .hero-container .article .box2 article a {
+    display: inline-block;
+    margin-top: 7px;
+  }
+  .form-container {
+    width: 97%;
+    margin: 0px auto;
+    padding: 40px 0;
+    display: -ms-grid;
+    display: grid;
+    place-items: center;
+  }
+  .footer-container {
+    width: 100vw;
+    background-color: #a55eea;
+    overflow-x: hidden;
+  }
+  .footer-container .footer-layout {
+    width: 100%;
+  }
+  .footer-container .footer-box {
+    width: 97%;
+    margin: 0px auto;
+    padding: 40px 0;
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 1fr;
+        grid-template-columns: 1fr;
+    -ms-grid-rows: 1fr 1fr;
+        grid-template-rows: 1fr 1fr;
+    place-items: center;
+    color: #fff;
+  }
+  .footer-container .footer-box p {
+    text-align: center;
+  }
+}
+/*# sourceMappingURL=style.css.map */
```

Comment:
add new line.

Reply:
actually my CSS is in style.scss where new line is already present, this file has been generated by scss extention to provide with css file version of the written scss